src/99-appendices/01-contributors.md

```diff
@@ -100,6 +100,7 @@ your name is not listed, please add it.
 -   Jean-Baptiste Poline 📖📢🤔🎨
 -   Vasudev Raguram 💻🎨📖🔧
 -   Ariel Rokem 📖
+-   Matt Sanderson 📖💻
```

Comment:
Please also add your github username to https://github.com/bids-standard/bids-specification/blob/master/CODEOWNERS